packages/web-twig/src/Twig/PropsExtension.php

```diff
@@ -91,4 +96,29 @@ public function renderClassProp(Environment $environment, array $classNames): st
             'classNames' => $filteredClassNames,
         ]);
     }
+
+    /**
+     * @param array<string, mixed> $props
+     */
+    public function renderStyleProp(Environment $environment, array $props): string
+    {
+        return $environment->render('@partials/styleProp.twig', [
+            'props' => $props,
+        ]);
+    }
+
+    /**
+     * @param array<string, mixed> $props
+     * @return array<string, mixed>
+     */
+    public function useStyleProps(array $props): array
+    {
+        $data = [];
+        if (isset($props['class'])) {
```

Comment:
```suggestion
        if (array_key_exists('class', $props)) {
```

https://www.php.net/manual/en/function.array-key-exists

Just the existence of the key is enough to raise the exception.